eng/automation/generate.py

```diff
@@ -127,15 +127,17 @@ def sdk_automation(input_file: str, output_file: str):
         logging.error("[GENERATE] Code generation failed. Unknown exception", exc_info=True)
         if packages and len(packages) == 1:
```

Comment:
just check, we should almost always have `len(packages) == 1` when the script got invoked?

Reply:
Yes.